sim/common/cata/damage_procs.go

```diff
@@ -70,7 +70,7 @@ func init() {
 		core.MakePermanent(core.MakeProcTriggerAura(&character.Unit, core.ProcTrigger{
 			Name:       "Electrical Charge Aura",
 			ActionID:   core.ActionID{ItemID: 68925},
-			Callback:   core.CallbackOnSpellHitDealt,
+			Callback:   core.CallbackOnSpellHitDealt | core.CallbackOnPeriodicHealDealt,
```

Comment:
this is meant to be periodic damage right

Reply:
Haha that's embarrassing... Thanks autocomplete 😅